doc/dev/developer_guide/testing_integration_tests/tests-integration-testing-teuthology-workflow.rst

```diff
@@ -9,28 +9,32 @@ Scheduling Test Run
 Getting binaries
 ****************
 
-To run integration tests using teuthology, you need to have Ceph binaries
-built for your branch. Follow these steps to initiate the build process -
-
-#. Push the branch to `ceph-ci`_ repository. This triggers the process of
-   building the binaries on jenkins CI.
-
-#. To confirm that the build process has been initiated, spot the branch name
-   at `Shaman`_. Little after the build process has been initiated, the single
-   entry with your branch name would multiply, each new entry for a different
-   combination of distro and flavour.
-
-#. Wait until the packages are built and uploaded to `Chacra`_, and the
```

Comment:
@neha-ojha This link goes to the Github page of the Chacra project and not to a running instance of Chacra. Is this what was intended? (I ask because I'm inclined to think that it wasn't.)

Reply:
I think it is fine to link the GH page that points to its README. Could add that the chacra url for a particular build can be queried from https://shaman.ceph.com/api/search/?status=ready&project=ceph.